### ISSUE TITLE
DependencyModel lacks Copyright assembly attribute

### ISSUE THREAD

**Issue by piotrpMSFT** [dotnet/core-setup#637]
Moved from https://github.com/dotnet/cli/issues/3829 on behalf of @dagood.

----

## Steps to reproduce

Look at properties for `Microsoft.Extensions.DependencyModel.dll` (1.0.0).
## Expected  behavior

Properties are like these (`System.Collections.dll` shown):

```
Product name = Microsoft® .NET Framework
Product version = 1.0.24212.01. Commit Hash: 9688ddbb62c04189cac4c4a06e31e93377dccd41
Copyright = © Microsoft Corporation.  All rights reserved.
```
## Actual behavior

Properties are these (`Microsoft.Extensions.DependencyModel.dll` shown):

```
Product name =
Product version = 1.0.0
Copyright =
```

(In CoreFX, the attributes that provide this data are [generated by Buildtools](https://github.com/dotnet/buildtools/blob/197567b/src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets#L57).)

**Comment by gkhanna79** [comment]
@schellap This is likely tied to moving Core-Setup rep to use BuildTools.

**Comment by gkhanna79** [comment]
@AlfredoMS Assigning this to you since you are working on the transition to build tools.

**Comment by eerhardt** [comment]
We also need to add the license file to both DependencyModel and PlatformAbstractions.

From a customer:

> Hello, The Microsoft.Extensions.DependencyModel package is missing the associated Licence. Could you add it please?

@Petermarcu @gkhanna79 - should this be fixed for 2.0?

**Comment by gkhanna79** [comment]
If we are doing the same thing for other binaries/packages, we should fix it for 2.0.
